drivers/system_pathways/app/models/system_pathways/report.rb

```diff
@@ -422,7 +422,7 @@ def allowed_states
               stay_length: stay_length,
               disabling_condition: en.enrollment.disabling_condition,
               relationship_to_hoh: en.enrollment.relationship_to_hoh,
-              chronic_at_entry: chronic_member[:chronic_status].present?,
+              chronic_at_entry: chronic_member.present? && chronic_member[:chronic_status].present?,
```

Comment:
Probably more inline with our style:
```suggestion
              chronic_at_entry: chronic_member.try(:[], :chronic_status).present?,
```